sources/game/Game.gd

```diff
@@ -47,9 +50,20 @@ func load_level(id: int) -> bool:
 	level_id = id
 	level = levels[id].instance()
 	add_child(level)
-
+	
 	assert(level.connect("level_end", self, "on_level_end") == OK)
 
+	if level is Level:
```

Comment:
It seems unnecessary, why not make it a child of Game in the editor and connect all signals there?

Reply:
Ok. There's issue here: I created a pause menu every time switch scene. Will fix it with this comment.

Reply:
After test, I found that we still need create new pause menu instance and add to level because:
- Level is newly instanced, too. I don't know if it's ok when we create too much instance. I'm not investigate godot memory management deeply
- Can't add an instance to multiple parent. I got error when create pause menu as child of Game and try to add to level again